index.js

```diff
@@ -15,7 +15,7 @@ class WebdriverAjax {
     this._wdajaxExpectations = [];
   }
 
-  before() {
+  before(config, capabilities, browser) {
```

Comment:
How about using underscore so we signal that we only use `browser` here.
```suggestion
  before(_, __, browser) {
```